docs/network/network-binding-plugin.md

```diff
@@ -492,3 +492,16 @@ The following points should be addressed in relation to migration support:
 > is started and plugging it back at the destination after the migration completes.
 > Future development will consider adding a new migration method to support
 > migration for such interfaces.
+
+## Compute Resource Overhead
+
+Some plugins may need additional resources to be added to the compute container of the virt-launcher pod.
+
+It is possible to specify compute resource overhead that will be added to compute containers of virt-launcher pods
+derived from virtual machines using the network binding plugin.
+
+> **Note**: At the moment, only memory overhead request is supported.
+
+For a network binding plugin to support compute resource overhead, the `computeResourceOverhead` field
+must be specified in the Kubevirt CR.
```

Comment:
Should we say the `computeResourceOverhead` should be specified with the required overhead?

Reply:
Since there is a link to the user-guide, it is implied that the configuration instructions will be there.